deliverables/README.md

```diff
@@ -11,17 +11,28 @@ Sign Up for Cloud9 and create a basic Rails project.
 ###Intro to Ruby
 ####Required
 
-What is an "interpreted" language?<br>
-What is IRB?<br>
-What are Objects?<br>
-What are Methods?<br>
-What are Classes?<br>
-What are Blocks?<br>
-What is an Array?<br>
-What is an Iterator?<br>
-What are hashes?<br>
-What is a library?<br>
-What is a gem?
+**What is an "interpreted" language?**<br>
+An interpreted language is where most implementations execute instructions directly without previously compiling a program into machine-language instructions. <br>
+**What is IRB?**<br>
+IRB stands for "interactive ruby" and allows the user to interactively execute Ruby expressions. <br>
+**What are Objects?**<br>
+Everything in Ruby is an object. ex: integers, characters, text, arrays, etc. <br>
+**What are Methods?**<br>
+Methods allow one object to interact with another. <br>
+**What are Classes?**<br>
+Classes allow the user to use the same method for multiple uses, it basically helps organize methods. <br>
+**What are Blocks?**<br>
+Blocks are instructions which may take arguments and return values. <br>
+**What is an Array?**<br>
+Array's are empty containers that are to be filled with objects or data. <br>
+**What is an Iterator?**<br>
+An iterator is a method that allows you to loop through all of the members of a collection. <br>
+**What are hashes?**<br>
+Hashes are indexed collections of objects of any type. <br>
```

Comment:
Incorrect.  Hashes are not indexed collections; they're labeled collections.  So instead of using an index like `1`, `22`, `0`, etc, you access hash items using a label like `"first_name"`, `:flavor`, etc.  

Example syntax to define a hash:  

``` ruby

a_hash = {
  "flavor" => "strawberry",
  "temperature_in_degrees_farenheight" => 34
}